command/secrets/init/params.go

```diff
@@ -44,10 +50,18 @@ func (ip *initParams) validateFlags() error {
 		return errInvalidParams
 	}
 
+	if ip.rawKeyType == "" {
+		return errInvalidParams
```

Comment:
This error is not appropriate for the key type, as it's related to directory initializations.

Please use a more appropriate error (create a new one) 🙏

Reply:
Added new error in 5c1ee77068408fb15fdddbcf3a0e77d14f46abc8